package/thingino-webui/files/var/www/x/tool-file-manager.cgi

```diff
@@ -45,7 +45,7 @@ dir=$(echo $dir | sed s#^//#/#)
 </thead>
 <tbody>
 <%
-lsfiles=$(ls -ALlp --group-directories-first $dir)
+lsfiles=$(ls -ALlpt --group-directories-first $dir)
```

Comment:
This policy applies all  file/directory listing including /. It is not right. It better to apply only on /mnt/mmcblk0p1/thingino.

Reply:
nope. no-nonsense alphabetical order stays.
hit the "end" button and it will scroll to the very bottom to see the last by date.

if you want an advanced file manager with custom sorting and such, think about a client-side implementation in javascript. but again, we have very little space for that.